### PR TITLE
[SSD1306] Added additional fonts 

### PR DESCRIPTION
**Pull request by jacquesoberto**
## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [x] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by jacquesoberto** [comment]
My first PR so be kind!

**Comment by josesimoes** [comment]
Hey @jacquesoberto ! please stay cool 😅 
Sorry for the comments cadence looking so daunting. Automation... 
Let me help with this and fix the initial comment. Then you can take it from there. 😉

**Review by Ellerbach — COMMENTED** [review]
Looks great! thanks a lot.
Would you mind adding a README with maybe a print of each letter of the lib on a picture? So developer will have an idea of how the font looks like?

**Comment by jacquesoberto** [comment]
@dotnet-policy-service agree

File: devices/Ssd13xx/Bitmap2Font/Bitmap2Font.cs [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    {
+        public Bitmap2Font()
+        {
+            InitializeComponent();
+        }
+
+        private Bitmap _myBitmap = new Bitmap(1, 1);
+        private string _imageIdentification = string.Empty;
+        private string _imageColorDepth = string.Empty;
+        private string _imageDimensions = string.Empty;
```
Comment (Ellerbach) [inline devices/Ssd13xx/Bitmap2Font/Bitmap2Font.cs:24]:
```suggestion
        private string _imageDimensions = string.Empty;
        private string _imageName = string.Empty;
```

**Review by Ellerbach — APPROVED** [review]
Looks good to me know. Thanks a lot for the extra kilometer to add the tool, document it, add some fonts! Looking forward to more contributions :-)

**Pull request merged**